src/libs/actions/PersonalDetails.js

```diff
@@ -50,6 +50,28 @@ function getDisplayName(login, personalDetail) {
     return fullName || userLogin;
 }
 
+/**
+ *
+ * @param {String} userAccountIDOrLogin
+ * @param {String} [defaultDisplayName] display name to use if user details don't exist in Onyx or if
+ *                                      found details don't include the user's displayName or login
+ * @returns {String}
+ */
+function getDisplayNameForTypingIndicator(userAccountIDOrLogin, defaultDisplayName = '') {
+    // Try to convert to a number, which means we have an accountID
+    const accountID = Number(userAccountIDOrLogin);
```

Comment:
Maybe check if it's a string instead? `Number('') === 0` which might lead to unexpected behavior.

Reply:
Ah... hmm... maybe just check `accountID && _.isNaN(accountID)` then?

Reply:
Or if `(!accountID) return;`

Reply:
Yeah i think i won't touch the logic much, there shouldn't be any account with accountID of 0, nor a login of `''` so we won't find anything which is fine 👍

Reply:
ok